network/discovery.go

```diff
@@ -177,6 +177,21 @@ func (d *discovery) isBootNode(id peer.ID) bool {
 
 	return false
 }
+
+func (d *discovery) addPeersToTable(nodes []string) error {
+	for _, node := range nodes {
+		info, err := StringToAddrInfo(node)
+		if err != nil {
+			continue
```

Comment:
Why not log the error?

Reply:
Commit-[link](https://github.com/0xPolygon/polygon-edge/pull/343/commits/12c1341ba166c39ce334c47b8943697cfabdfccd)